src/c/BuscaBinaria.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+
+#define TRUE 1
+#define FALSE 0
+
+int binarySearch(int array[], int value, int len)
+{
+  int mid = len / 2;
+  int newArray[mid];
+  int result = FALSE;
+
+  if (value > array[mid] && len >= 1)
+  {
+    for (int i = 0; i < mid; i++)
```

Comment:
Só uma dica/sugestão aqui, por mais que tenha apenas uma linha dentro do `for` (isso também serve para `if` ou qualquer bloco de código), grande parte dos desenvolvedores dá preferência por incluir as chaves pq, apesar de ser mais verboso, isso torna o código menos propenso a erros, achei uma discussão falando sobre isso aqui: https://softwareengineering.stackexchange.com/a/16530